### DISCUSSION TITLE
Odd continue behavior, am I missing something?

### DISCUSSION THREAD

**Discussion by gctypo** [cmus/cmus#1175] [Troubleshooting]
I'm a bit confused about the behavior of a few settings, specifically with how they work with playlists. I'm getting some unexpected behavior and I honestly can't tell if I've encountered a bug or just a misunderstanding of these settings:

```
continue (currently: true)
  Continue playing after current track finishes.

continue_album (currently: true)
  Continue playing next album after current album finishes.

follow (currently: true)
  If enabled, always select the currently playing track on track change.

stop_after_queue (currently: false)
  Stop playback when end of play queue is reached.
```

Aside from `follow`, they all *seem* pretty easy to understand in the context of playing from playlist. But I want to double-check I'm not misunderstanding so I'm not reporting a full-on issue for something stupid like "turn off `setting_name_here`".

The behavior I'm getting is when starting Cmus and playing a playlist with repeat (playlist repeat, not track repeat) and shuffle, continue doesn't seem to work. Continue enabled, start a song, wait for it to finish. It advances to the next (randomly-selected) song, but does not start it. If I press play it starts normally, will play to its end, and then advance but not start again.

Disabling and re-enabling continue does not fix this issue, but if I toggle some of the other settings I listed it starts working. The issue does not occur every startup, and if I get it to work again it stays fixed for the duration of that session (and usually the next few sessions).

Is one of these settings causing this odd behavior, or is this a bug?

**Comment by pgaskin** [comment]
Can you paste the entire autosave file?
